environments/staging/public.yml

```diff
@@ -71,8 +71,10 @@ localsettings:
   ALLOWED_HOSTS:
     - j2mestaging.commcarehq.org
     - testserver
+    - sanjay1.commcarehq.org
   ANALYTICS_DEBUG: True
   ANALYTICS_LOG_LEVEL: "debug"
+  ALTERNATE_HOSTS: sanjay1.commcarehq.org
```

Comment:
If this were a list of domains, would the commands below need to change?

Reply:
I am testing this and will update

Reply:
we have to add list of domains with space separated by.

eg:  ALTERNATE_HOSTS: sanjay1.commcarehq.org sanjay2.commcarehq.org

Reply:
I was thinking in particular of the certbot commands, which have lines like this:
```
-d {{SITE_HOST}} -d {{ NO_WWW_SITE_HOST }} -d {{ ALTERNATE_HOSTS }}
```
I imagine that'd render as
```
-d www.commcarehq.org -d commcarehq.org -d sanjay1.commcarehq.org sanjay2.commcarehq.org
```
When I suspect the command would require a `-d` before `sanjay2.commcarehq.org`